test/unit/animation.js

```diff
@@ -332,6 +332,34 @@
     }, 1000);
   });
 
+  QUnit.test('animate with list of values', function(assert) {
+    var done = assert.async();
+    assert.expect(52)
```

Comment:
what is this?

Reply:
This confirms the number of assertions should be run in this test, so we make sure we are running all the async tests before finishing.

Reply:
I had no idea this was a feature.
So this test has 52 assertion?